framework/auth/cas.py

```diff
@@ -208,15 +213,23 @@ def get_logout_url(*args, **kwargs):
     """
     return get_client().get_logout_url(*args, **kwargs)
 
+
 def get_profile_url():
     """Convenience function for getting a profile URL for a user.
     """
     return get_client().get_profile_url()
 
+
```

Comment:
Nitpicky, but we tell `flake` to ignore the PEP8 requirement of having two blank lines between methods. Not a blocker